pandas/io/tests/test_excel.py

```diff
@@ -1218,6 +1241,25 @@ def test_datetimes(self):
 
             tm.assert_series_equal(write_frame['A'], read_frame['A'])
 
+    # GH7074
+    def test_bytes_io(self):
+        bio = BytesIO()
```

Comment:
awesome. can you add a small section to the excel docs (io.rst) showing writing to bytes?